### ISSUE TITLE
Can't send file

### ISSUE THREAD

**Issue by DCsunset** [uTox/uTox#1253]
### Overview

1. uTox version: 0.16.1
2. Operating system: Ubuntu 18.04
3. Desktop Environment/Window Manager (Linux/BSD only): GNOME

### Issue

When clicking at the "Send File" button, nothing happens.
But the screenshot button works.

The output from terminal in verbose mode after each click at "Send File":
```
XLIB:         ButtonEvent: 0 1
XLIB:         ButtonEvent: 256 1
```

**Comment by DCsunset** [comment]
Possibly duplicate [#1129](https://github.com/uTox/uTox/issues/1129)

**Comment by redmanmale** [comment]
Probably was fixed in #1009.
Please try the latest (0.17.0) version.
